mesonbuild/modules/pkgconfig.py

```diff
@@ -136,6 +138,26 @@ def generate(self, state, args, kwargs):
         pub_reqs = mesonlib.stringlistify(kwargs.get('requires', []))
         priv_reqs = mesonlib.stringlistify(kwargs.get('requires_private', []))
         conflicts = mesonlib.stringlistify(kwargs.get('conflicts', []))
+
+        def parse_variable_list(stringlist):
+            reserved = ['prefix', 'libdir', 'includedir']
+            variables = []
+            for var in stringlist:
+                try:
+                    k, v = var.split('=')
```

Comment:
I'm not sure I understand what you mean. My point was `foo=bar=baz` is probably valid.